doc/content/verification_and_validation/val-2a.md

```diff
@@ -0,0 +1,103 @@
+
+The source term in the model is described as a normal distribution instead of a piecewise function from TMAP4. The source term of deuterium versus the coordinate is defined as:
+
+\begin{equation} \label{eq:normal_distribution}
+S = \frac{1.5}{\sigma \sqrt(2 \pi) \exp \left( -0.5 * ((x - \mu ) / \sigma)^2 \right)},
+\end{equation}
+
+where $\sigma$ is the characteristic width of the normal distribution, and $\mu$ is the depth of the normal distribution from the upstream side. The comparison between the normal distribution from TMAP8 and piecewise function from TMAP4 is shown in [val-2a_normal_distribution]. The normal distribution has the similar distribution with the piecewise function, and also provide a smooth distribution, which is close to the real situation.
+
+!media figures/val-2a_normal_distribution.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=val-2a_normal_distribution
+       caption= Comparison between the normal distribution from TMAP8 and piecewise function from TMAP4. The area of both functions are normalized as 1.
+
+The pressures on the upstream and downstream sides are close to the vacuum situation, and have less impact for the recommendation on two sides. Thus, TMAP8 ignore the impact of pressure on the boundary conditions to simplify the model. The recommendation is described as:
+
+\begin{equation} \label{eq:recommendation_ignore_Pressure}
+J = A K_r C^2.
+\end{equation}
+
+## Case and Model Parameters
+
+The beam flux on the upstream side of the sample during the experiment are presented in [val-2a_flux_and_pressure_TMAP4]. Other case and model parameters in TMAP8 are listed in [val-2a_set_up_values_TMAP4]. Some of the parameters are directly leveraged from [!cite](anderl1985tritium,longhurst1992verification,ambrosek2008verification), but others were adapted to better match the experimental data.
+
+!table id=val-2a_flux_and_pressure_TMAP4 caption=Values of beam flux on the upstream side of the sample during the experiment [!citep](longhurst1992verification).
+| time (s)      | Beam flux (atom/m$^2$/s)     |
+| ---------     | ---------------------------- |
+| 0 - 5820      | 4.9$\times 10^{19}$          |
+| 5820 - 9056   | 0                            |
+| 9056 - 12062  | 4.9$\times 10^{19}$          |
+| 12062 - 14572 | 0                            |
+| 14572 - 17678 | 4.9$\times 10^{19}$          |
+| 17678 - 20000 | 0                            |
+
+!alert warning title=Typo in [!cite](longhurst1992verification)
+The times of starting and stopping the beam from TMAP4 in [!cite](longhurst1992verification) are not accurate, TMAP8 uses the times from [!cite](anderl1985tritium) directly to get better fit on results.
+
+!table id=val-2a_set_up_values_TMAP4 caption=Values of material properties.
```

Comment:
A a reference column and add the reference used for each parameter value whenever you found it somewhere else.